src/main.rs

```diff
@@ -56,9 +58,23 @@ fn main() -> ! {
     // Set up the LED (B12).
     let mut led = gpiob.pb12.into_push_pull_output(&mut gpiob.crh);
 
-    // Set up serial communication on pins A9 (Tx) and A10 (Rx), with 115200 baud.
-    let usart_pins = (gpioa.pa9.into_alternate_push_pull(&mut gpioa.crh), gpioa.pa10);
-    let mut protocol = SerialProtocol::new(dp.USART1, usart_pins, &mut afio, &mut rcc.apb2, clocks);
+    // Set up USB communications
+    let usb_pin_d_plus = gpioa.pa12;
+    let usb_pin_d_minus = gpioa.pa11;
+
+    let usb = Peripheral { usb: dp.USB, pin_dm: usb_pin_d_minus, pin_dp: usb_pin_d_plus };
+
+    let usb_bus = UsbBus::new(usb);
+    let serial = SerialPort::new(&usb_bus);
+
+    let usb_dev = UsbDeviceBuilder::new(&usb_bus, UsbVidPid(0x16c0, 0x27dd))
+        .manufacturer("tonari")
+        .product("tonari hardware controller")
+        .serial_number("12345")
```

Comment:
Why not just 1, and should this be unique per device for any reason?

Reply:
From what I've observed, it'll append a `1`, `2`, `3`, etc. as you plug more devices in.

No reason to make it unique per device though.